fp.pretty/TSOT-23-6_B2.8x2.9X1.45mm_P0.95mm.kicad_mod

```diff
@@ -0,0 +1,36 @@
+(module TSOT-23-6_B2.8x2.9X1.45mm_P0.95mm (layer F.Cu) (tedit 5F3D0FAD)
+  (attr smd)
+  (fp_text reference REF** (at -2.5 -0.5 270) (layer F.SilkS)
+    (effects (font (size 1 1) (thickness 0.15)))
+  )
+  (fp_text value TSOT-23-6_B2.8x2.9X1.45mm_P0.95mm (at 1 -3.5) (layer F.Fab)
+    (effects (font (size 1 1) (thickness 0.15)))
+  )
+  (fp_line (start 1.75 -2.3) (end 1.75 2.3) (layer F.SilkS) (width 0.12))
+  (fp_line (start 1.75 2.3) (end -0.95 2.3) (layer F.SilkS) (width 0.12))
+  (fp_line (start -1.75 -2.3) (end 1.75 -2.3) (layer F.SilkS) (width 0.12))
+  (fp_line (start 1.5 -2.05) (end 1.5 2.05) (layer F.CrtYd) (width 0.05))
+  (fp_line (start -1.75 1.5) (end -1.75 -2.3) (layer F.SilkS) (width 0.12))
+  (fp_line (start -0.95 2.3) (end -1.75 1.5) (layer F.SilkS) (width 0.12))
+  (fp_line (start -1.5 2.05) (end -1.5 -2.05) (layer F.CrtYd) (width 0.05))
+  (fp_line (start 1.5 2.05) (end -1.5 2.05) (layer F.CrtYd) (width 0.05))
+  (fp_text user REF** (at 0 -0.5 90) (layer F.Fab)
+    (effects (font (size 0.7 0.7) (thickness 0.125)))
+  )
+  (fp_line (start -1.5 -2.05) (end 1.5 -2.05) (layer F.CrtYd) (width 0.05))
+  (fp_line (start -1.65 -2.17) (end 1.61 -2.17) (layer F.Fab) (width 0.12))
+  (fp_line (start 1.61 -2.17) (end 1.61 2.17) (layer F.Fab) (width 0.12))
+  (fp_line (start 1.61 2.17) (end -1.64 2.17) (layer F.Fab) (width 0.12))
+  (fp_line (start -1.64 2.17) (end -1.65 -2.17) (layer F.Fab) (width 0.12))
+  (pad 2 smd rect (at 0 1.2) (size 0.6 1.2) (layers F.Cu F.Paste F.Mask))
+  (pad 3 smd rect (at 0.95 1.2) (size 0.6 1.2) (layers F.Cu F.Paste F.Mask))
+  (pad 4 smd rect (at 0.95 -1.2) (size 0.6 1.2) (layers F.Cu F.Paste F.Mask))
+  (pad 6 smd rect (at -0.95 -1.2) (size 0.6 1.2) (layers F.Cu F.Paste F.Mask))
+  (pad 5 smd rect (at 0 -1.2) (size 0.6 1.2) (layers F.Cu F.Paste F.Mask))
+  (pad 1 smd rect (at -0.95 1.2) (size 0.6 1.2) (layers F.Cu F.Paste F.Mask))
+  (model "C:/Users/anique alamgir/Downloads/User Library-SOT23-6-1.step"
```

Comment:
This should be a path in `${KISYS3DMOD}`, see here: https://kicad-pcb.org/libraries/klc/F9.3/

Reply:
Done